.gitlab-ci.yml

```diff
@@ -84,7 +99,10 @@ get_agent_version:
 
 .build:
   stage: build
-  except: [tags]
+  rules:
+    - if: $CI_COMMIT_TAG != null
+      when: never
```

Comment:
❓ Question :
What does `CI_COMMIT_TAG` refer to again ? does this mean we need at least a PR tag to run the CI ? or if it's a commit tag it means that a simple `git commit -m <msg> && git push` won't trigger the pipeline ?

Reply:
`The commit tag name. Available only in pipelines for tags.`
So if a pipeline is created with `git tag tagname && git push origin tagname` this job will not be added to the pipeline

Reply:
Aah right good 👍